src/client/pages/OfferNew/Introduction/index.tsx

```diff
@@ -59,55 +77,33 @@ export const Introduction: React.FC<Props> = ({
   refetch,
   onCheckoutOpen,
 }) => {
-  const [sidebarIsSticky, setSidebarIsSticky] = React.useState(false)
-  const ref = React.useRef<HTMLDivElement>(null)
-
-  useDocumentScroll(() => {
-    const distanceToTop =
-      ref.current !== null
-        ? ref.current.getBoundingClientRect().top
-        : Math.pow(10, 10)
-
-    if (distanceToTop <= 96) {
-      setSidebarIsSticky(true)
-    } else {
-      setSidebarIsSticky(false)
-    }
-  })
-
-  const hasDataCollection =
-    !isBundle(offerData) && !!offerData.quotes[0].dataCollectionId
+  const [hasHeroImageLoaded, setHasHeroImageLoaded] = useState(false)
 
   return (
     <Section>
       <Wrapper>
-        <Container>
-          <DesktopLeadingContainer>
-            <Usps />
-            {hasDataCollection && (
-              <ExternalInsuranceProvider
-                dataCollectionId={offerData.quotes[0].dataCollectionId || ''}
-                offerData={offerData}
-              />
-            )}
-          </DesktopLeadingContainer>
-          <MobileLeadingContainer>
-            {hasDataCollection && (
-              <ExternalInsuranceProvider
-                dataCollectionId={offerData.quotes[0].dataCollectionId || ''}
-                offerData={offerData}
-              />
-            )}
-            <Usps />
-          </MobileLeadingContainer>
-          <Sidebar
-            ref={ref}
-            sticky={sidebarIsSticky}
-            offerData={offerData}
-            refetchOfferData={refetch}
-            onCheckoutOpen={onCheckoutOpen}
+        <Hero>
+          <HeroImage
+            alt="laptop grip"
+            onLoad={() => setHasHeroImageLoaded(true)}
+            hasLoaded={hasHeroImageLoaded}
+            src="/new-member-assets/landing/laptop_grip_small.jpg"
+            sizes="100vw"
+            srcSet="
+          /new-member-assets/landing/laptop_grip_small.jpg 1600w,
+          /new-member-assets/landing/laptop_grip_medium.jpg 2200w"
```

Comment:
In my head when picturing a Hero it makes most sense when having the content of the Hero inside it as well. In this case it would mean renaming the `Wrapper` to `Hero` instead. Not a big deal and the Wrapper was there from before, just something I got stuck on 😄